orchestration/dagster_orchestration/hca_orchestration/pipelines/load_hca.py

```diff
@@ -34,7 +35,7 @@
 dev_mode = ModeDefinition(
     name="dev",
     resource_defs={
-        "beam_runner": preconfigure_resource_for_mode(k8s_dataflow_beam_runner, "dev"),
+        "beam_runner": preconfigure_resource_for_mode(dataflow_beam_runner, "dev"),
```

Comment:
This still runs the transformation jobs in dataflow, but I didn't want to muck with k8s locally so I've switched this to just dispatch the dataflow jobs via sbt.